src/service/repo/signedMessageRepo.ts

```diff
@@ -89,30 +89,39 @@ async function sendSignedMessage(
   }
 }
 
-async function getAllSignedChannels() {
+async function getAllSignedChannels(timeStamp: string | undefined) {
+  const url = timeStamp
+    ? `${baseUrl.getAllSignedChannels}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllSignedChannels;
   try {
-    const response = await api.get(baseUrl.getAllSignedChannels);
-
+    const response = await api.get(url);
     if (response.status === 200) {
+      console.log('SUCCESS URL', url);
       return Promise.resolve(response.data?.data);
     }
 
     return Promise.reject(response.data?.status);
   } catch (e) {
+    console.log('FAILED URL', url);
     console.log(e);
     return Promise.reject(e);
   }
 }
 
-async function getAllSignedPostNotifications() {
+async function getAllSignedPostNotifications(timeStamp: string | undefined) {
+  const url = timeStamp
+    ? `${baseUrl.getAllSignedPostNotifications}?last_fetch_date=${timeStamp}`
+    : baseUrl.getAllSignedPostNotifications;
   try {
-    const response = await api.get(baseUrl.getAllSignedPostNotifications);
+    const response = await api.get(url);
     if (response.status === 200) {
+      console.log('SUCCESS URL', url);
       return Promise.resolve(response.data?.data);
     }
 
     return Promise.reject(response.data?.status);
   } catch (e) {
+    console.log('FAILED URL', url);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider removing the console.log statements in the `getAllSignedPostNotifications` function.

```diff
- 118:       console.log('SUCCESS URL', url);
...
- 124:     console.log('FAILED URL', url);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->